book/website/afterword/glossary.md

```diff
@@ -129,14 +129,14 @@ CRediT Taxonomy
 
 ```{glossary}
 
-Data repository
-  See repository.
+Data Repository
+    A long-lived place on the internet where resources (data, software, publications or anything else) can be stored and accessed. Often Data Repositories provide long term preservation and persistent identifiers for the research objects stored.
```

Comment:
I might use the term "storage space" as opposed to long-lived, for greater clarity. Also, perhaps Data Repositories should be lowercase?

Reply:
And is that the only feature? Aren't data repositories much bigger in scale as well?

Reply:
And what's the difference between a database and a data repository? That might be another definition worth adding!

Reply:
I had another go at the definitions! Thanks for jumping in :)